flo-workflow/src/test/java/com/spotify/flo/context/EvalContextTest.java

```diff
@@ -0,0 +1,60 @@
+/*-
+ * -\-\-
+ * Flo Workflow Definition
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.flo.context;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.spotify.flo.EvalContext;
+import com.spotify.flo.Task;
+import com.spotify.flo.TaskOperator;
+import com.spotify.flo.context.InstrumentedContext.Listener;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+
+public class EvalContextTest {
+
+  private Listener listener = mock(Listener.class);
```

Comment:
Prefer using [`@MockitoJunitRunner`](https://static.javadoc.io/org.mockito/mockito-core/2.6.8/org/mockito/junit/MockitoJUnitRunner.html) and `@Mock` annotations